src/scripts/bootstrap_trigger/bootstrap_trigger.py

```diff
@@ -235,20 +232,22 @@ def main():
                 dataset_uri=args.diff_s3_uri,
                 aws_session=aws_session,
                 columns=[args.diff_parquet_field])
-        already_processed_records = parquet_dataset[args.diff_parquet_field]
+        records_needing_processing = synapse_df.drop(
```

Comment:
Just so I'm understanding, the default isn't taking the union and you have to specify both diffs. 

1. Why don't we take the union by default?
2. Do you envision more diff fields to be added?

Reply:
> Why don't we take the union by default?

The union is taken "by default" if the user specifies two parquet datasets. Are you suggesting we remove the ability to only diff upon a single parquet dataset? What would it look like to take the union by default?

This is the current behavior:

* Specify no parquet datasets, and everything on Synapse is submitted
* Specify one parquet dataset, and the diff is taken upon that parquet dataset
* Specify two parquet datasets, and we submit the union of each parquet dataset diff'd upon Synapse.

> Do you envision more diff fields to be added?

Only the metadata and assessment data are required files, so it doesn't make sense to add others. You would submit for (re)processing all the records which didn't include a specific optional file (?) I can't think of a plausible use case.

Reply:
Ahh thanks for the explanation!  I misunderstood the default behavior. Approving